DCA632FFFE2828A0.json

```diff
@@ -0,0 +1,11 @@
+{
+	"gateway_conf": {
+		"gateway_ID": "DCA632FFFE2828A0",
+		"servers": [ { "server_address": "eu1.cloud.thethings.network", "serv_port_up": 1700, "serv_port_down": 1700, "serv_enabled": true } ],
+		"ref_latitude": 12.972442,
+		"ref_longitude": 77.580643,
+		"ref_altitude": 920,
+		"contact_email": "",
```

Comment:
please add an email

Reply:
arroncarson1@gmail.com

Reply:
Not you, the author of the PR. And not in a comment, as a commit. Stop posting non-sense @Dndptvvxcn